ref/general/application-hardening.adoc

```diff
@@ -61,25 +65,50 @@ To alleviate this risk, use wildcards to secure servlets wherever possible.
 If that is not appropriate, carefully double check your `web.xml` file before deployment.
 
 When you specify authorization constraints for servlets, ensure that you either list no methods or carefully list all the methods for a servlet, including GET, POST, PUT, HEAD, etc.
-It's likely that you must list all the methods if you are using multiple constraints.
+If you are using multiple constraints, list all the methods.
 
-In the following secure example, endpoints in the web application are unreachable by default.
-If a new servlet is added, you must add a security constraint for the new URL so that anyone can use it:
+Specify the `<deny-uncovered-http-methods>` element] in the `web.xml` file.
+When the `<deny-uncovered-http-methods>` element is specified, the container denies any uncovered HTTP methods that aren't listed in the security constraints for a URL pattern that best matches the request URL.
```

Comment:
I'm not sure what this means- 

`...the container denies any uncovered HTTP methods that aren't listed in the security constraints for a URL pattern that best matches the request URL.`

it's not clear what `for a URL pattern that best matches the request URL.` modifies. Is it security constraints? or HTTP methods?